src/CORS/src/Microsoft.AspNetCore.Cors/Infrastructure/ICorsAttribute.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Cors.Infrastructure
+{
+    /// <summary>
+    /// A marker interface which can be used to identify CORS metdata.
+    /// </summary>
+    public interface ICorsAttribute
```

Comment:
`internal interface`?

Reply:
It needs to be public so if anyone else is searching for CORS metadata they can use the marker interface to find the most significant metadata.

Reply:
`ICorsPolicyMetadata`?